graspy/inference/latent_distribution_test.py

```diff
@@ -174,6 +238,54 @@ def _embed(self, A1, A2):
 
         return X1_hat, X2_hat
 
+    def _estimate_correction_variances(self, X_hat, Y_hat, pooled=False):
+        # TODO it is unclear whether using pooled estimator provides more or
+        # less power. this should be investigated. should not matter under null.
+        N, d_X = X_hat.shape
+        M, d_Y = Y_hat.shape
+        if N == M:
+            X_sigmas = np.zeros((N, d_X, d_X))
+            Y_sigmas = np.zeros((M, d_Y, d_Y))
+        elif N > M:
```

Comment:
is it possible to write in such a way that you always operate on the bigger one if not pooled, as opposed to having the two branches of this if/else be basically the same w/ args flipped?

Reply:
i.e. X could always be assigned the bigger graph if not equal, or something like that

Reply:
i had internal debate about between elegance and readability when writing this. having two branches makes the symmetry more apparent, whereas abusing the order of the graphs is more concise. i feel like the two are about the same in terms of a probability of introducing a bug, since one has twice more code to mess up in, and the other has the ordering to keep track of. i went with the readability, but it was a 55-45 call, and since you feel the other way, i am happy to change it.